client/src/components/new-request/quick-search-results.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <MainContainer id="quick-search-results-container">
+    <template v-slot:container-header>
+      <v-col cols="auto" class="h6 py-0 mt-1">
+        You are searching for a name for a
+        {{ entityText === ' BC Corporation' && location.text === ' BC' ? '' : ' ' + location.text }}
+        {{ entityText }}
+      </v-col>
+    </template>
+    <template v-slot:content>
+      <v-row no-gutters class="pt-3">
+        <NameInput/>
+      </v-row>
+      <v-row justify="center" class="ma-0 pa-0">
+        <v-col cols="12" class="quick-search-title">
+          <b>Quick Search Results</b>
+        </v-col>
+      </v-row>
+      <v-row v-if="namesList.length" justify="center" class="ma-0 pa-0">
+        <v-col cols="12" class="quick-search-info">
+          We found <b>{{namesList.length}}</b> names using these, or similar, words.
+        </v-col>
+        <v-col cols="12" class="quick-search-info">
+          Names cannot be too similar to existing names within the same business category.
+          Try and request a unique name.
+        </v-col>
+        <v-col cols="12" class="names-list">
+          <QuickSearchNames/>
+        </v-col>
+      </v-row>
+      <v-row v-else justify="center">
+        <v-col cols="12" class="quick-search-info">
+          We did not find any names using these exact or similar words within the same business category.
+        </v-col>
+      </v-row>
+      <v-row justify="center">
+        <v-col cols="auto" class="pb-0 pr-1">
+          <v-btn id="search-again-button" :disabled="nameChanged" @click="searchAgain">Search Again</v-btn>
+        </v-col>
+        <v-col cols="auto" class="pb-0 pr-1">
+          <v-btn class="continue-search-button" color="primary" outlined @click="detailedSearch">
+            Continue to Detailed Analysis
```

Comment:
Generally it's better if the text is on the same line as the > and < of the button element, or within a span on its own line, to facilitate testing (ie, you can select the exact button text without a newline before or after it).
We can fix this later.

Reply:
okay sounds good, I split it up because the line was too long